content/en/docs/reference/command-line-tools-reference/kubelet.md

```diff
@@ -390,7 +390,7 @@ kubelet [flags]
        <td colspan="2">--enable-cadvisor-json-endpoints</td>
     </tr>
     <tr>
-      <td></td><td style="line-height: 130%; word-wrap: break-word;">Enable cAdvisor json /spec and /stats/* endpoints. (default true)</td>
+      <td></td><td style="line-height: 130%; word-wrap: break-word;">Enable cAdvisor json /spec and /stats/* endpoints. (default false) (DEPRECATED: will be removed in a future version)</td>
```

Comment:
@wawa0210 , Thanks for this pull request.
This update looks good.
```
./kubelet --help | grep "enable-cadvisor-json-endpoints"
--enable-cadvisor-json-endpoints         Enable cAdvisor json /spec and /stats/* endpoints. (DEPRECATED: will be removed in a future version)
```

Reply:
This is generated documentation, right - does this fix need to go upstream?